libbs/api/decompiler_interface.py

```diff
@@ -137,6 +148,8 @@ def _init_gui_plugin(self, *args, **kwargs):
         return None
 
     def shutdown(self):
+        self.config.save()
```

Comment:
It's a good idea to save here, but you should also save the moment someone connects to a `Client` because it's very likely the user kills BinSync before we get to this nice shutdown.